README.md

```diff
@@ -1,78 +1,34 @@
 # PlasmaPy
 
-[![Build Status](https://travis-ci.org/PlasmaPy/PlasmaPy.svg?branch=master)](https://travis-ci.org/PlasmaPy/PlasmaPy) [![Coverage Status](https://coveralls.io/repos/github/PlasmaPy/PlasmaPy/badge.svg?branch=master)](https://coveralls.io/github/PlasmaPy/PlasmaPy?branch=master)
-
-A community developed Python package for plasma physics.
-
-## Project Status
-
-PlasmaPy is in an early stage of development.  The goals of this project are described in our [vision statement](https://github.com/PlasmaPy/PlasmaPy/blob/master/vision_statement.md) and an earlier conference poster:
-
-* Murphy, Nicholas A, Huang, Yi-Min, & PlasmaPy Community. (2016, October). PlasmaPy: beginning a community developed Python package for plasma physics. Zenodo. http://doi.org/10.5281/zenodo.163752
-
-We recently created a guide on [contributing to PlasmaPy](https://github.com/PlasmaPy/PlasmaPy/blob/master/CONTRIBUTING.md), which also contains instructions on how to join our email list.  New contributors are very welcome!  
-
-If you have any questions, please contact Nick Murphy at <namurphy@cfa.harvard.edu>, Yi-Min Huang at <yiminh@princeton.edu>, or Drew Leonard at <andy.j.leonard@gmail.com>.
+[![Build Status](https://travis-ci.org/PlasmaPy/PlasmaPy.svg?branch=master)](https://travis-ci.org/PlasmaPy/PlasmaPy)
+[![Coverage Status](https://coveralls.io/repos/github/PlasmaPy/PlasmaPy/badge.svg?branch=master)](https://coveralls.io/github/PlasmaPy/PlasmaPy?branch=master)
+
+A community developed Python 3.6+ package for plasma physics in 
+the early stages. PlasmaPy intends to be for plasmas what
+[Astropy](https://github.com/astropy/astropy) is for astronomy - a 
+collection of functionality commonly used and shared between plasma physicists 
+and researchers globally, running within and leveraging the open source 
+scientific Python ecosystem. The goals of this project are better described in our
```

Comment:
Instead of "better", could we say "more thoroughly"?

Reply:
Done!